cvat-core/src/api-implementation.ts

```diff
@@ -85,9 +86,16 @@ export default function implementAPI(cvat) {
         await serverProxy.server.logout();
     };
 
-    cvat.server.advancedAuthentication.implementation = async () => {
-        const result = await serverProxy.server.advancedAuthentication();
-        return result;
+    cvat.server.socialAuthentication.implementation = async () => {
+        const result: SocialAuthMethodsRawType = await serverProxy.server.socialAuthentication();
+        const methods = [];
```

Comment:
More functional style would be:

```suggestion
        return Object.entries(result).map(([provider, value]) => new SocialAuthMethod({ ...value, provider }))
```